exercises/practice/queen-attack/test.8th

```diff
@@ -0,0 +1,61 @@
+"queen-attack.8th" f:include
+needs exercism/test
+with: test
+13 tests
+
+"Queen with a valid position"
+    ( 2 2 create )
+    not-null?
```

Comment:
I've defined the `create` word such that it allows the user to choose their own representation (array, complex number, or whatever). The main downside of this is that I can't use one of the existing words to check for a specific value. I added a `non-null?` word to the `test` lib. Do you think that makes sense?

Reply:
In the `test` lib before this one, IIRC, I had a `non-null?` word. It makes sense to me.